src/index.scss

```diff
@@ -1,5 +1,6 @@
 body {
```

Comment:
Since we're editing this file, can we add `@import '~bootstrap/scss/bootstrap';` to the top of this file?